web/js/index.js

```diff
@@ -278,7 +278,44 @@ fetch(`/api/elements${window.location.search}`, { headers: { 'Accept': 'applicat
                     displayError(`Error while requesting element parts: ${err}`);
                 }
             }
+            else if (elem.elementType === 'ASSEMBLY'){
+                const child = document.createElement('option');
+                child.setAttribute('href', `${window.location.search}&gltfElementId=${elem.id}`);
+                child.innerText = `Assembly - ${elem.name}`;
+                $elemSelector.appendChild(child);
+            }
         }
     }).catch((err) => {
         displayError(`Error while requesting document elements: ${err}`);
     });
+
+const dropdown = document.getElementById('elem-selector');
+const selectedOptionText = document.getElementById('selected-option');
+
+dropdown.addEventListener('change', function() {
```

Comment:
As far as I know, we don't need to write this function.
The select HTML element gives us the functionality that is implemented in this event listener.

Reply:
Done.